File: atp/src/dominio/Consulta.java
package dominio;

public class Consulta {
	private String court;
	private int roundImp;
	private int bestOf;
	private int player0;
	private int player1;

	public void setCourt(String court) {
 	   this.court = court;
    }

    public String getCourt() {
        return court;
    }

    public void setRoundImp(int roundImp) {
 	 	this.roundImp = roundImp;
    }

    public int getRoundImp() {
        return roundImp;
    }

    public void setBestOf(int bestOf) {
 		this.bestOf = bestOf;
    }

    public int getBestOf() {
        return bestOf;
    }

    public void setPlayer0(int player0) {
 		this.player0 = player0;
 	}

    public int getPlayer0() {
    	return player0;
    }

    public void setPlayer1(int player1) {
    	this.player1 = player1;
    }

    public int getPlayer1() {
        return player1;
    }

    public Consulta(String court, int roundImp, int bestOf, int player0, int player1){
    	this.court = court;
    	this.roundImp = roundImp;
    	this.bestOf = bestOf;
    	this.player0 = player0;
    	this.player1 = player1;
    }

    @Override
    public String toString() {
        return getCourt() + "," + getRoundImp() + "," + getBestOf() + "," + getPlayer0() + 
        "," + getPlayer1() + ",?" + "\n%\n%";
    }
}

File: mk_2/app/src/dominio/consulta.py
class Consulta():
    
    def __init__(self, court, round_imp, best_of, player0, player1):
        self._court = court
        self._round_imp = round_imp
        self._best_of = best_of
        self._player0 = player0
        self._player1 = player1
    
    def __str__(self):
        return f"{self._court}, {self._round_imp}, {self._best_of}, {self._player0}, {self._player1}"
    
    # Ver como hacer que esto funcione
    @property
    def court(self):
        return self._court

    @court.setter
    def court(self, court):
        self._court = court


